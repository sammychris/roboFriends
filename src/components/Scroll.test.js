import { shallow } from "enzyme";
import Scroll from "./Scroll";

const wrapper = shallow(<Scroll />);

it("expect to render scroll", () => {
  expect(wrapper).toMatchSnapshot();
});
