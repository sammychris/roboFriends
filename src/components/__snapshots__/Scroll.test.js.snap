// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render scroll 1`] = `ShallowWrapper {}`;
